docs/merkle-sum-trie.md

```diff
@@ -1,5 +1,7 @@
 # Sparse Merkle Sum Trie (smst)
 
+TODO_IN_THIS_PR(@Olshansk): Document the new `count` addition.
```

Comment:
Suggestion to make this an issue instead as you'll have to update all digest documentation ie. digest = `hashSize+8+8` in bytes to account for the change